dn.go

```diff
@@ -108,114 +142,154 @@ func (d *DN) String() string {
 	return strings.Join(rdns, ",")
 }
 
+// Remove leading and trailing spaces from the attribute type and value
+// and unescape any escaped characters in these fields
+// Remove leading and trailing spaces from the attribute type and value
+// and unescape any escaped characters in these fields
```

Comment:
Accidentally doubled the comment?

Reply:
Thank you for pointing this out @johnweldon ❤️